### PR TITLE
Added springboot function image

### PR DESCRIPTION
**Pull request by amariucaitheodor**
This PR adds a new example to the vHive ensemble of function images. Features Springboot, Java and gRPC.

### CODE REVIEW

File: function-images/springboot/.gitignore [hunk 1]
```diff
@@ -0,0 +1,33 @@
+HELP.md
```
Comment (ustiugov) [inline function-images/springboot/.gitignore:1]: please delete

File: function-images/springboot/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# vHive-bench-java
```
Comment (ustiugov) [inline function-images/springboot/README.md:1]: Can you add info where you took the basis of the code from?

File: function-images/springboot/src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (ustiugov) [inline function-images/springboot/src/main/resources/application.properties:1]: is it supposed to be empty?

**Review by ustiugov — CHANGES_REQUESTED** [review]
needs minor changes

**Comment by ustiugov** [comment]
please add this image build to the [workflow](https://github.com/ease-lab/vhive/blob/7242618923e6a2d4730f02d38cf99173ff573047/.github/workflows/build_docker_images.yml#L16)

**Comment by ustiugov** [comment]
Few more fixes that are necessary by convention of our tool:
* the function's server must be listening to port 50051
* the function need to return different responses, depending on the input. Here how it works in Python functions:
```
    def SayHello(self, request, context):
        if request.name == "record":
            msg = 'Hello, %s!' % responses[0]
        elif request.name == "replay":
            msg = 'Hello, %s!' % responses[1]
        else:
            msg = 'Hello, %s!' % request.name

        return helloworld_pb2.HelloReply(message=msg)

File: function-images/springboot/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+
+This project was bootstrapped using Spring Initializr at start.spring.io
```
Comment (ustiugov) [inline function-images/springboot/README.md:3]: typo Initializr? the link must be formatted as a link

**Review by ustiugov — CHANGES_REQUESTED** [review]
Please add the image name to [TestAllFunctions](https://github.com/ease-lab/vhive/blob/7242618923e6a2d4730f02d38cf99173ff573047/vhive_test.go#L297)

File: vhive_test.go [hunk 5]
```diff
@@ -294,6 +294,7 @@ func TestAllFunctions(t *testing.T) {
 		"vhiveease/cnn_serving:var_workload",
 		"vhiveease/rnn_serving:var_workload",
 		"vhiveease/lr_training:var_workload",
+		"vhiveease/springboot:var_workload",
```
Comment (ustiugov) [inline vhive_test.go:297]: don't forget to push the workload with this tag to dockerhub

File: function-images/springboot/README.md (hunk 4 above)
Comment (amariucaitheodor) [reply function-images/springboot/README.md]: Not a typo

Comment (ustiugov) [reply function-images/springboot/README.md]: then please add this to configs/.wordlist* file so that the spellchecker is happy

**Review by ustiugov — APPROVED** [review]

**Pull request merged**